### ISSUE TITLE
Should ripple update be necessary after a ripple init?

### ISSUE THREAD

**Issue by KevM** [FubuMvcArchive/ripple#89]
I ripple-ized a project with ripple init and noticed that the references of the .csproj was not updated. I assumed that the ripple init would set it all up.

The _ripple.config_ had the dependencies described. I later did a ripple update and that fixed the references in the csproj to be non-versioned with a HintPath to the correct directory under packages.
### myproject.csproj

``` patch
-    <Reference Include="log4net, Version=1.2.11.0, Culture=neutral, PublicKeyToken=669e0ddf0bb1aa2a, processorArchitecture=MSIL">
-      <HintPath>..\packages\log4net.2.0.0\lib\net40-full\log4net.dll</HintPath>
+    <Reference Include="FubuCore">
+      <HintPath>..\packages\FubuCore\lib\FubuCore.dll</HintPath>
     </Reference>
-    <Reference Include="StructureMap, Version=2.6.4.0, Culture=neutral, PublicKeyToken=e60ad81abae3c223, processorArchitecture=MSIL">
-      <HintPath>..\packages\structuremap.2.6.4.1\lib\net40\StructureMap.dll</HintPath>
+    <Reference Include="log4net">
+      <HintPath>..\packages\log4net\lib\net40-full\log4net.dll</HintPath>
+    </Reference>
+    <Reference Include="StructureMap">
+      <HintPath>..\packages\structuremap\lib\net40\StructureMap.dll</HintPath>
```

**Comment by jmarnold** [comment]
Ripple init wasn't intended to convert a solution from nuget. I think we're going to need a new command for that.

**Comment by jmarnold** [comment]
Relates to #95

**Comment by jmarnold** [comment]
This will be fixed in 2.3.
